docs/content/getting-started/CLI.md

```diff
@@ -0,0 +1,33 @@
+---
+category: getting-started
+categoryindex: 1
+index: 3
+---
+
+# Command Line Arguments
+
+## Example Command
+
+When running the CLI tool from the command line, the bare minimum you need to provide is the path to the project file(s) you want to analyze.
```

Comment:
This can be a bit misleading. It sounds like something will happen simply by downloading the tool and pointing it to a project, but you actually need to provide an analyzer.

The default argument will only have an effect if you:
- Use Paket in the old mode with that exact default folder name.
- Already have a compatible analyzer.

This scenario seems quite unlikely at this point. It might be better to mention later that if you omit the `--analyzers-path`, it defaults to `packages/analyzers`.